components/eventing-controller/config/helm/kustomization.yaml

```diff
@@ -0,0 +1,33 @@
+# Adds namespace to all resources.
+namespace: kyma-system
+
+# Value of this field is prepended to the
+# names of all resources, e.g. a deployment named
+# "wordpress" becomes "alices-wordpress".
+# Note that it should also match with the prefix (text before '-') of the namespace
+# field above.
+#namePrefix: eventing-
+
+#generatorOptions:
+#  disableNameSuffixHash: true
+
+# Include CRDs because they are not part of eventing-controller helm chart.
+bases:
+  - ../crd
+
+resources:
```

Comment:
(nit) Since we do not have a deploy option for EPP, and according to the AC we need make targets for EPP as well, should we add the EPP resources too?

Reply:
The docker image for EPP will be updated by KO in this make target. Rest of the resources for EPP would be already on the cluster.